superset-frontend/src/views/CRUD/data/database/DatabaseModal.tsx

```diff
@@ -421,7 +449,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
                       />
                     </div>
                   </StyledInputContainer>
-                  <StyledInputContainer>
+                  <StyledInputContainer className="mb-0">
```

Comment:
I know I added this mb-0, :) but I like @rusackas' suggestion of using emotion here. Can we create a const called `no-margin-bottom` or something like that? He put together a https://codesandbox.io/s/emotion-playground-forked-7z07l?file=/src/index.js. Thanks!

Reply:
I like this a lot better too! Fixed in [`this commit`](https://github.com/apache/superset/pull/14293/commits/542d52c9db122a5b39d92f1ae955c5636731014f)